src/Factory.php

```diff
@@ -46,7 +46,7 @@ class Factory
     public function __construct(LoopInterface $loop = null, ConnectorInterface $connector = null)
     {
         if ($connector === null) {
-            $connector = new Connector($loop);
+            $connector = new Connector();
```

Comment:
Argument should not be omitted if explicit loop is given.

Reply:
:+1:

Reply:
This should probably be `new Connector(array(), $loop)`, no? (https://github.com/reactphp/socket/pull/264)

Reply:
You're right, I overlooked that:sweat_smile: